README.md

```diff
@@ -113,3 +114,17 @@ val response = myEventsClient.createCustomObjectRecord(request)
 println(response.id) // HubSpot ID
 println(response.properties["name"]) // Party #2022
 ```
+### Associate a contact to an existing company with default label
+```kotlin
+
+val associationRequest = AssociationRequest(
+    fromObjectType = CONTACT, // contact, company, deal, etc
+    fromObjectId = 1, // HubSpot ID of the contact
+    toObjectType = COMPANY, // contact, company, deal, etc
+    toObjectId = 2 // HubSpot ID of the company
+)
+
+val associationResponse = associationClient.createDefaultAssociation(associationRequest)
```

Comment:
Can you please add one line with `val associationClient = AssociationClient(hubspotClient)`?